### ISSUE TITLE
Very Unstable

### ISSUE THREAD

**Issue by KamuiAlmighty** [joffnerd/botw-trainer#143]
Compared to the previous version, this current trainer is very unstable. Weapon durability values change randomly after every in-game loading screen and rupees constantly reset to zero when selling/buying items at a merchant. In addition, the trainer crashes frequently, and when I try to reconnect to my Wii U, it crashes the game as well.

And what's more? The previous version of the trainer does not work with the previous setup (using CosmoCortney's codehandler) anymore, so I was forced to upgrade to this version and change my entire Homebrew/TCPGecko setup.

TL;DR: trainer works, but it's very unstable and unreliable. I preferred the previous version.

**Comment by Tarog** [comment]
Please delete this comment since it is not a question,
you are just complaining 
if it isnt working its not the trainers fault or joffners its the falut of your connection, setup, and or wii u

the trainer is working perfectly fine for me

**Comment by KamuiAlmighty** [comment]
First and foremost, do not ask me to delete my comment. If a site moderator or joffnerd himself finds my comment obtrusive, offensive, or otherwise inappropriate, they will delete it or request to have it deleted. Do not concern yourself with things that do not pertain to you.

Secondly, if one produces a program such as this, criticism (constructive or otherwise) is inevitable, and, I would imagine, welcomed. After all, it's the only way to improve it. If you contributed to the project, grow a thicker skin and learn to deal with feedback properly. If you didn't, please be quiet, as this comment wasn't intended for you. Again, mind your own business.

Also, please delete your comment since it is not a question. You are just complaining about a complaint.

**Comment by joffnerd** [comment]
TBH i am not a  software developer, if it doesn't work then i recommend using what does. It's always going to track bully's tcp gecko so you will have to deal with that being updated. Also, you can't connect until you have loaded a save.

I doubt i will update  it anytime soon tbh. I have  never had stability issues, it either works or doesn't.

While I appreciate feedback, just telling me its unreliable or unstable doesn't really help.

**Comment by KamuiAlmighty** [comment]
I appreciate the response, Joff. Big fan, by the way. The trainer's amazing. I did some troubleshooting over the past couple days with some pointers listed on the gbatemp.net forums, and it's working much better now. However, there's still the issue of when I disconnect the trainer from the Wii U, it crashes my game. It's not a huge issue (since I can just turn the Wii U off and then close the trainer), but it might be something worth looking into. As I said previously, this didn't happen with the previous version.

I understand you won't be updating it for the time being, but again, thank you for your time and the effort you put into this.

Also, if there's a somewhere I could make a donation to you or the project, do let me know. Since I don't know squat about coding or software development, I would like to at least support those who do.

**Comment by joffnerd** [comment]
Thanks for the kind words, didn't mean to be  short initially.

I just made a quick paypal button, never taken donations before but feel free to try it out:

https://joffnerd.s3.amazonaws.com/donate.html
